defaults/keymaps-common.toml

```diff
@@ -0,0 +1,157 @@
+# --------------------------------- General --------------------------------------------
+
+[[keymaps]]
+key = "F1"
+command = "palette.command"
+
+[[keymaps]]
+key = ":"
+command = "palette.command"
+mode = "n"
+
+[[keymaps]]
+key = "space"
+command = "toggle_code_lens"
+mode = "nv"
+
+# --------------------------------- Basic editing ---------------------------------------
+
+[[keymaps]]
+key = "alt+up"
+command = "move_line_up"
+mode = "i"
+
+[[keymaps]]
+key = "alt+down"
+command = "move_line_down"
+mode = "i"
+
+# ------------------------------------ Multi cursor -------------------------------------
+
+[[keymaps]]
+key = "alt+I"
+command = "insert_cursor_end_of_line"
+mode = "i"
+
+# ------------------------------------ ------------ -------------------------------------
+
+[[keymaps]]
+key = "ctrl+m"
+command = "list.select"
+when = "list_focus"
+
+[[keymaps]]
+key = "enter"
+command = "list.select"
+when = "list_focus"
+
+[[keymaps]]
+key = "ctrl+p"
+command = "list.previous"
+when = "list_focus"
+
+[[keymaps]]
+key = "up"
+command = "list.previous"
+when = "list_focus"
+
+[[keymaps]]
+key = "ctrl+n"
+command = "list.next"
+when = "list_focus"
+
+[[keymaps]]
+key = "down"
+command = "list.next"
+when = "list_focus"
+
+[[keymaps]]
+key = "o"
+command = "list.expand"
+when = "list_focus"
+mode = "n"
+
+[[keymaps]]
+key = "/"
+command = "palette.line"
+mode = "n"
+
+[[keymaps]]
+key = "esc"
+command = "palette.cancel"
+when = "palette_focus"
+
+[[keymaps]]
+key = "esc"
+command = "code_actions.cancel"
+when = "code_actions_focus"
+
+[[keymaps]]
+key = "ctrl+b"
+command = "left"
+mode = "i"
+
+[[keymaps]]
+key = "ctrl+f"
+command = "right"
+mode = "i"
+
+[[keymaps]]
+key = "right"
+command = "right"
+
+[[keymaps]]
+key = "left"
+command = "left"
+
+[[keymaps]]
+key = "up"
+command = "up"
+when = "!list_focus"

```

Comment:
`up` and `down` only had the `when = "!list_focus"` in the macOS keymap. I don't know which version of this is the intended, so I picked the macOS one and added the `when`.